course/functions.php

```diff
@@ -85,3 +85,26 @@ function course_register_block_patterns_category() {
 
 add_action( 'init', 'course_register_block_patterns_category' );
 
+add_filter(
+	'theme_lesson_templates',
+	'course_theme_filter_single_lesson_template_for_sensei_learning_mode',
+	11,
+	3
+);
+
+function course_theme_filter_single_lesson_template_for_sensei_learning_mode( $page_templates, $theme, $post ) {
```

Comment:
This function is missing a DocBlock.

Reply:
Thanks! Updated it here https://github.com/Automattic/themes/pull/7265/commits/9015d270c250378f71c842018b6aff0ec2cdab77